app/lib/primer/classify.rb

```diff
@@ -236,6 +236,10 @@ def extract_value(memo, key, val, breakpoint)
           memo[:classes] << "#{key.to_s.dasherize}#{breakpoint}-#{val.to_s.dasherize}"
         end
       end
+
+      def force_system_arguments?
+        Rails.application.config.primer_view_components.force_system_arguments
```

Comment:
Can we explicitly set this value to false by default? We wouldn't want this to accidentally be turned on in production.